ci/Jenkinsfile

```diff
@@ -120,9 +120,7 @@ pipeline {
                             def error_logs_message = ""
                             dir("${HOMEgfs}/sorc") {
                                 try {
-                                    sh(script: './build_all.sh -kgu')  // build the global-workflow executables for GFS variant (UFS-wx-model, WW3 pre/post executables)
-                                    sh(script: './build_ww3prepost.sh -w > ./logs/build_ww3prepost_gefs.log 2>&1')  // build the WW3 pre/post processing executables for GEFS variant
-                                    sh(script: './build_ufs.sh -w -e gefs_model.x > ./logs/build_ufs_gefs.log 2>&1')  // build the UFS-wx-model executable for GEFS variant
+                                    sh(script: './build_all.sh gfs gefs gsi gdas')  // build the global-workflow executables
```

Comment:
Wouldn't the `gdas` also build `gsi_monitor` and `gsi_utils`

Reply:
Yes, but `gsi` is still needed to build the `gsi_enkf`.

Reply:
Perhaps `gdas_builds` should only include `gdas` and not `gsi_utils gsi_monitor`.

Reply:
Maybe, but I think the GDASApp-based DA still makes use of some gsi-utils and gsi-monitor executables.  @RussTreadon-NOAA @CoryMartin-NOAA is that still the case?

Reply:
it might. but the point here is that this invocation is going to build `gsi_monitor` and `gsi_utils` twice.  
Could we run into race conditions?

Reply:
Ah, I see.  No, it will only build once.  `build_all.sh` uses associative arrays, so `build[gsi_monitor]=yes` will be assigned, so it will only build once.

Reply:
Ok. thanks I understand now from the other comment, that an intersection is performed and all duplicate entries are removed.